### ISSUE TITLE
Incorrectly parse opacity

### ISSUE THREAD

**Issue by Semigradsky** [omgovich/colord#33]
```ts
colord('rgb(29 164 192 / 95%)')
colord('rgb(29 164 192 / 0.95)')
// r {
//   parsed: { r: 29, g: 164, b: 192, a: 1 },
//   rgba: { r: 29, g: 164, b: 192, a: 1 }
// }
```

But works with old syntax (with commas):
```ts
colord('rgb(29, 164, 192, 0.95)')
// r {
//   parsed: { r: 29, g: 164, b: 192, a: 0.95 },
//   rgba: { r: 29, g: 164, b: 192, a: 0.95 }
// }
```

**Comment by omgovich** [comment]
Hi, thanks for the issue! 
I didn't implement all CSS notations yet but will do that soon. Probably in a few days.
Will let you once it's done 🤝

**Comment by omgovich** [comment]
Fixed in v1.3.1 🚀
